src/App.tsx

```diff
@@ -85,6 +85,7 @@ function App() {
             {globalState.wizardStep === TMCRFinalStep && globalState.wizardOptions.length === 1 && <Button type="button" onClick={e => dispatch({ type: 'ADD_TMCR' })}>
               Add second TMCR
             </Button>}
+            {(globalState.wizardStep !== TMCRFinalStep || (globalState.wizardOptions.length === 1 || (globalState.wizardOptions.length === 2 && globalState.tmcrIndex === 1))) && 
```

Comment:
@RobertPorterfield - This is just trying to hide the button on all screens except the Final Step or the First TMCR if there is  a second TMCR, right?  Would it be clearer to write it that way !(globalState.wizardOptions.length === 2 && globalState.wizardStep === TMCRFinalStep && globalState.tmcrIndex === 0)?